Update screenshots and steps for new UI architecture (#1234)
astronomer/docs: astro/configure-deployment-resources.md

## Scheduler resources

> The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks once dependencies have been met. By adjusting the **Scheduler Count** slider in the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Resources**.

Reviewer: ```suggestion
The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks when the dependencies are met. By adjusting the **Scheduler Count** slider in the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Resources**.
```

> The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks once dependencies have been met. By adjusting the **Scheduler Count** slider in the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Resources**.

Reviewer 2: ```suggestion
The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html) is responsible for monitoring task execution and triggering downstream tasks once dependencies have been met. By adjusting the **Scheduler Count** slider in the **Configuration** tab of the Cloud UI, you can configure up to 4 schedulers, each of which will be provisioned with the AU specified in **Resources**.
```

1. Log in to the [Cloud UI](https://cloud.astronomer.io) and select a Workspace.
2. Select a Deployment.
3. Select one of the following options to adjust the settings of a Worker Queue or a Scheduler:

    - To change the settings of an existing worker queue, click **Edit** in the **Worker Queues** area, adjust the settings, and then click **Update Queue**.
    - To change scheduler resources, click **Configuration** and then click **Edit Configuration**.

4. Edit the Deployment resource settings. For more information about these settings, review the content in this topic.

> 5. Click **Update Deployment**.

Reviewer 2: Doesn't look like there's an `**Update Deployment** button anymore. Should we just split these sections up into editing scheduler vs. worker queue?

For worker queue, it's **Update queue* and for scheduler it's just **Update**

> 5. Click **Update Deployment**.

Author: ```suggestion
5. Click **Update**.
```

2. Select a Deployment.

> 3. Based on the resource setting you want to change, click one of the following buttons:

Reviewer: ```suggestion
3. Select one of the following options to adjust the settings of a Worker Queue or a Scheduler:
```

>     - To create a worker queue or change an existing worker queue, click **Worker Queues**.

Reviewer: ```suggestion
    - To change the settings of an existing worker queue, click **Edit*, adjust the settings, and then click **Update Queue**.
```

>     - To change scheduler resources, click **Configuration** and then click **Edit Configuration**.

Reviewer: ```suggestion
    - To change the settings of an existing Scheduler, click **Edit** in the **Scheduler Settings** area,  adjust the settings, and then click **Update Scheduler**.
```
